test/controllers/private_messages_controller_test.rb

```diff
@@ -0,0 +1,16 @@
+require 'test_helper'
+
+class PrivateMessagesControllerTest < ActionController::TestCase
+  setup do
+
+    @ticket = tickets(:problem)
+    @reply = replies(:solution)
+
+    sign_in users(:alice)
+  end
+
+  test 'create private message' do
+    put :create, private_message: {ticket_id: @ticket.id, message: 'test message'}
```

Comment:
Line is too long. [82/80]<br>Space inside { missing.<br>Space inside } missing.<br>Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.